README.md

```diff
@@ -25,11 +25,35 @@ pnpm add -g @sern/cli@latest
 
 When you install the CLI, you can use our commands with **sern** prefix.
 
-`sern <command> (opt)<flag>`
+```
+Usage: sern [options] [command]
+
+Welcome to Sern!
+If you're new to Sern, run sern init for an interactive setup to your new bot project!
+
+If you have any ideas, suggestions, bug reports, kindly join our support server: https://discord.gg/xzK5fUKT4r
+
+Options:
+  -V, --version      output the version number
+  -h, --help         display help for command
+
+Commands:
+  init [options]     Quickest way to scaffold a new project
+  plugins [options]  Install plugins from https://github.com/sern-handler/awesome-plugins
+  extra              Easy way to add extra things in your sern project
+  help [command]     display help for command
+
+```
 
 ## Setting Up Your Project
 
-#### TODO
+Run `sern init (-y)` for an interactive setup on a brand new project using our framework. <br>
+Adding the `-y` flag sets up project as default.
+
+## Installing Plugins
+
+Sern runs on your plugins. Contribute to the [repository](https://github.com/sern-handler/awesome-plugins) and then install the plugins via our cli! <br>
+`sern plugins (-n | --name) "Name of plugin"`
```

Comment:
I don't think sern plugins provide name option yet
nor i plan to add one, _maybe_ until we have a lot of plugins
i already provide _autocomplete search_ and _multi select box_ for plugins so they can type names after doing `sern plugins`